strength_percentiles.py

```diff
@@ -0,0 +1,92 @@
+# Script Name	: strength_percentiles.py
+# Author		: Kadeem Nibbs
+# Created		: 7/16/2017
+# Last Modified	:
+# Version		: 1.0
+
+# This program will define a program to scrape powerlifting websites for
+# lifters' squat, bench, deadlift numbers.  People who use this
+# program will be able to enter their own lift numbers and see where their
+# percentile rank among the competitive strength training population
+
+import urllib2
+from bs4 import BeautifulSoup
+import re
+
+# Results for New York State Powerlifting meet
+
+quote_page = 'http://meets.revolutionpowerlifting.com/results/2016-meet-results/ny-states/'
+# contains list of tuples, each tuple (squat, bench, deadlift, total),
+# represents lifts made by individual of group corresponding to dictionary key
+lifts = {'all': [],
+        'amateur': [],
+        'professional': [],
+        'men': [],
+        'women': [],
+        'raw': [],
+        'equipped': []}
+
+page = urllib2.urlopen(quote_page)
+soup = BeautifulSoup(page, 'html.parser')
+
+# list of HTML table row tag objects, each corresponds to an individual's
+# performance at this meet
+meet_results_table = soup.find_all('tr')
+
+gender_column = 1
+professional_status_column = 3
+equipment_column = 5
+squat_column = 15 # column containing lifters' squat numbers, counting from 0
+bench_column = 17 # ...
+deadlift_column = 19 # ...
+
+for result in meet_results_table: # information for one lifter
```

Comment:
It's unclear what you want to do for each row of this table. Could use a comment before the loop saying what you're trying to do.